contracts/Staking.sol

```diff
@@ -267,9 +267,11 @@ contract Staking is IStaking, Initializable, OwnableUpgradeable, Constants {
         _delegateToIndexer(_indexer, _indexer, _amount);
     }
 
+    //TODO: stake the commission to indexer
     function stakeCommission(address _indexer, uint256 _amount) external {
         require(msg.sender == settings.getRewardsDistributer(), 'G003');
-        //_addDelegation(_indexer, _indexer, _amount);
+        emit DelegationAdded(_indexer, _indexer, _amount);
+        _startUnbond(_indexer, _indexer, _amount);
```

Comment:
Would be good if we can identify the `unbond` type between `undelegation` and `commissionRate`, otherwise, on the frontend and subql project, user will not be able to see the source of unbonding